Services/IotHub/IotHubConnectionStringManager.cs

```diff
@@ -172,7 +175,7 @@ private async Task UsePreprovisionedIotHubAsync()
             try
             {
                 this.ValidateConnectionString(this.config.IoTHubConnString);
-                await this.TestIoTHubReadPermissionsAsync(this.config.IoTHubConnString);
+                this.TestIoTHubReadPermissionsAsync(this.config.IoTHubConnString);
```

Comment:
ditto

Reply:
because the function that replaced the deprecated function is no longer asynchronous and so we don't need to wait on it